ios/ConverseNotificationExtension/Xmtp/Messages.swift

```diff
@@ -85,7 +78,7 @@ func handleV3Message(xmtpClient: XMTP.Client, envelope: XMTP.Xmtp_MessageApi_V1_
               }
               // We replaced decodedMessage.senderAddress from inboxId to actual address
               // so it appears well in the app until inboxId is a first class citizen
-              if let senderProfileSocials = await getProfile(account: xmtpClient.address, address: decodedMessage.senderInboxId) {
+              if let senderProfileSocials = await getInboxIdProfile(account: xmtpClient.address, inboxId: decodedMessage.senderInboxId) {
```

Comment:
asx we come across these methods, they need to be renamed to findThingByInboxId/findThingByAddress